dockerfiles/docker-compose.yml

```diff
@@ -109,6 +109,16 @@ services:
       - RTD_STRIPE_SECRET
       - RTD_STRIPE_PUBLISHABLE
       - RTD_DJSTRIPE_WEBHOOK_SECRET
+      - RTD_SOCIALACCOUNT_PROVIDERS_GITHUB_CLIENT_ID
+      - RTD_SOCIALACCOUNT_PROVIDERS_GITHUB_SECRET
+      - RTD_SOCIALACCOUNT_PROVIDERS_GITLAB_CLIENT_ID
+      - RTD_SOCIALACCOUNT_PROVIDERS_GITLAB_SECRET
+      - RTD_SOCIALACCOUNT_PROVIDERS_BITBUCKET_OAUTH2_CLIENT_ID
+      - RTD_SOCIALACCOUNT_PROVIDERS_BITBUCKET_OAUTH2_SECRET
+      - RTD_SOCIALACCOUNT_PROVIDERS_BITBUCKET_CLIENT_ID
+      - RTD_SOCIALACCOUNT_PROVIDERS_BITBUCKET_SECRET
```

Comment:
We aren't actively using these (are deprecated by allauth too). https://github.com/readthedocs/readthedocs.org/issues/11162

```suggestion
```